BSP/Inc/Interrupt_Priorities.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
```

Comment:
Could you add comments here that follow the coding styles guide?

Reply:
fixed